daemon/smartcard.c

```diff
@@ -168,8 +196,28 @@ smartcard_cb_start_container(int fd, unsigned events, event_io_t *io, void *data
 				out.has_wrapped_key = true;
 				out.wrapped_key.len = keylen;
 				out.wrapped_key.data = key;
+				out.container_uuid = mem_strdup(uuid_string(container_get_uuid(
+								startdata->container)));
+
+				out.has_token_type = true;
+				switch (container_get_token_type(startdata->container)) {
```

Comment:
if container_token_t is used in smartcard_tokentype_to_proto()
-> remove switch case and directly assign 
out.token_type = smartcard_tokentype_to_proto(container_get_token_type(startdata->container);